source/mir/random/engine/pcg.d

```diff
@@ -434,9 +429,7 @@ mixin template specific_stream(Uint)
     ///
     Uint inc_ = default_increment!Uint;
     ///
-    @property Uint increment()() { return inc_; }
-    ///
-    @property void increment()(Uint u) { inc_ = u;}
+    alias increment = inc_;
```

Comment:
For some reason `@property Uint increment()() { return inc_; }` was causing `save` not to work. Changing it to a non-template property function was sufficient for `save` to work but I went a step further and replaced the property with an alias.

Reply:
Related bug report https://issues.dlang.org/show_bug.cgi?id=21744